Assets/Plugins/LeapMotion/Modules/Hands/Scripts/HandBinder/HandBinderAutoBinder.cs

```diff
@@ -49,17 +43,48 @@ public static class HandBinderAutoRigger {
             handBinder.boundHand.fingers[4].boundBones = AssignUnityBone(pinkyBones);
             handBinder.boundHand.wrist = AssignBoundBone(wrist);
             handBinder.boundHand.elbow = AssignBoundBone(Elbow);
+
             if(wrist != null && Elbow != null) {
                 handBinder.elbowLength = (wrist.position - Elbow.position).magnitude;
             }
 
             EstimateWristRotationOffset(handBinder);
+            CalculateElbowLength(handBinder);
+            CheckForAssignedBones(ref handBinder);
```

Comment:
Check if this is needed, probably not required here as handBinder is not being reassigned in CheckForAssignedBones(...)

Reply:
Turns out the entire function was not needed, nice spot